qa/suites/fs/volumes/tasks/volumes/test/fwdauth.yaml

```diff
@@ -0,0 +1,11 @@
+teuthology:
+  postmerge:
+    - if not is_fuse() then reject() end
+tasks:
+- exec:
+    mon.a:
+      - ceph fs set cephfs balance_automate false
+- cephfs_test_runner:
+    fail_on_skip: false
+    modules:
+      - tasks.cephfs.test_misc.TestForwardAuth
```

Comment:
It's weird to put this in fs:volumes. That should only be testing the mgr `volumes` plugin. 

Why not put this in `fs:functional`?

Reply:
Sure.

Reply:
This yaml is still in `fs/volumes`?

Reply:
Okay, I forgot to do that.